### PR TITLE
fix code and note blocks in dualtor pages

### PR DESCRIPTION
**Pull request by aaaaaaaalex**
Product Version(s):
mdx syntax bugfix applies to all versions.
New documentation applies only to 3.20EP2 and later.

Issue:
[PMREQ-629](https://tigera.atlassian.net/browse/PMREQ-629)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed 

[PMREQ-629]: https://tigera.atlassian.net/browse/PMREQ-629?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: calico-enterprise/networking/configuring/dual-tor.mdx [hunk 1]
```diff
@@ -475,6 +473,7 @@ startup prior to receiving BGPPeer and BGPConfiguration resources from the datas
 EarlyNetworkingConfiguration will be superseded by any BGPPeer or BGPConfiguration
 resources after successful startup.
 
+For a full description of the EarlyNetworkingConfiguration resource, see [here](../../reference/resources/earlynetworkconfiguration.mdx)
```
Comment (ctauchen) [inline calico-enterprise/networking/configuring/dual-tor.mdx]:
```suggestion
For a full description, see [Early Network Configuration](../../reference/resources/earlynetworkconfiguration.mdx)
```

Comment (ctauchen) [reply calico-enterprise/networking/configuring/dual-tor.mdx]: What's the difference between EarlyNetworkingConfiguration (line 473) and EarlyNetworkConfiguration?

File: calico-enterprise/reference/resources/earlynetworkconfiguration.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+# Early Network Configuration
+
+Early network configuration supplies labels and AS numbers to apply to each
+{{prodname}} node, as well as peering and other network configuration to use during node
+startup prior to receiving BGPPeer and BGPConfiguration resources from the datastore.
+Early network configuration will be superseded by any BGPPeer or BGPConfiguration
+resources after successful startup.
+
+For a detailed guide on how to set up dual ToR, see [here](../../networking/configuring/dual-tor.mdx).
```
Comment (ctauchen) [inline calico-enterprise/reference/resources/earlynetworkconfiguration.mdx]:
```suggestion
For a detailed guide on how to set up dual ToR, see [Deploy a dual ToR cluster](../../networking/configuring/dual-tor.mdx).
```

Comment (ctauchen) [reply calico-enterprise/reference/resources/earlynetworkconfiguration.mdx]: Always good practice to have specific anchors.

File: calico-enterprise/reference/resources/earlynetworkconfiguration.mdx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+:::note
+Early network configurations are intended for use only when {{prodname}} is being run in "early networking" mode, i.e. when operating a "dual ToR" networking configuration. If you are not running a dual-plane cluster, this is probably not what you are looking for!
+For configuring BGP networking under normal circumstances (when {{prodname}} is running), see [BGPConfiguration](bgpconfig.mdx), [BGPPeer](bgppeer.mdx), and [BGPFilter](bgpfilter.mdx).
+:::
```
Comment (ctauchen) [inline calico-enterprise/reference/resources/earlynetworkconfiguration.mdx]: Consider moving this information into the top paragraph. If it's really only for dual-tor, you can say that in the first description.

File: calico-enterprise/reference/resources/earlynetworkconfiguration.mdx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+| nodeIPFromDefaultRoute                | Toggles legacy behavior, where {{prodname}} early networking would use the src IP of a default route to identify which ConfigNode to use.                                | true, false     | false          |
+| unconditionalDefaultRouteProgramming  | Toggles legacy behavior, where {{prodname}} early networking would unconditionally program a default route to a ToR, regardless of whether one existed prior to startup. | true, false     | false          |
+
+### ConfigStableAddress
+| Field                                 | Description                                                     | Accepted Values   | Schema Default |
+| ------------------------------------- | --------------------------------------------------------------- | ----------------- | -------------- |
+| address | IP address which {{prodname}} should provision during earlynetworking as the "stable" address | IP Address string | ""             |
```
Comment (ctauchen) [inline calico-enterprise/reference/resources/earlynetworkconfiguration.mdx]: during earlynetworking --> during early networking

If that's the established wording. Sounds a bit funny to my ear.

File: calico-enterprise/reference/resources/earlynetworkconfiguration.mdx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+### ConfigPeering
+| Field        | Description                                                                                               | Accepted Values   | Schema Default |
+| ------------ | --------------------------------------------------------------------------------------------------------- | ----------------- | -------------- |
+| peerIP       | The IP address of the ToR                                                                                 | IP Address string | ""             |
+| peerASNumber | The AS number of the ToR. When left as 0, {{prodname}} early-networking will use its own node's AS number | integer           | 0              |
```
Comment (ctauchen) [inline calico-enterprise/reference/resources/earlynetworkconfiguration.mdx]: early networking

**Review by ctauchen — COMMENTED** [review]
Thanks for this! A few comments for you.

File: calico-enterprise/networking/configuring/dual-tor.mdx (hunk 1 above)
Comment (aaaaaaaalex) [reply calico-enterprise/networking/configuring/dual-tor.mdx]: EarlyNetworkingConfiguration doesn't exist - typo!

**Comment by aaaaaaaalex** [comment]
Feedback is all implemented now, reviewdog is finding English typos within YAML, safe to ignore.

**Comment by ctauchen** [comment]
LGTM, thanks!

**Pull request merged**